modules/validations.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-classes-per-file */
+
+const error = document.querySelector('#error');
+
+const showmessage = (test, msg) => {
+  if (test) {
+    error.textContent = msg;
+    return test;
+  }
+  error.textContent = msg;
+  return test;
+};
+
+export default function validations(input, msg, input2, msg2) {
```

Comment:
- [ ] please make use of ES6 which included arrow functions instead of the function keyword.